exporter/kafkaexporter/README.md

```diff
@@ -30,12 +30,15 @@ The following settings can be optionally configured:
 - `logs`
   - `topic` (default = otlp\_logs): The name of the Kafka topic to which logs will be exported.
   - `encoding` (default = otlp\_proto): The encoding for logs. See [Supported encodings](#supported-encodings).
+  - `topic_from_metadata_key` (default = ""): The name of the metadata key whose value should be used as the message's topic. Useful to dynamically produce to topics based on request inputs. It takes precedence over `topic_from_attribute` and `topic` settings.
```

Comment:
Can you please update the "Destination Topic" section at the end of the README? Maybe the details you added on these lines can be moved there, and we can copy the short phrased used elsewhere: `See [Destination Topic](#destination-topic) below for more details.`

Reply:
oh good point. Updated in the last commit.